Task 9/script.js

```diff
@@ -9,3 +9,22 @@ budget: number
 Metodas: 
 wasExpensive() - jeigu filmo "budget" yra daugiau nei 100 000 000 mln USD, tada grąžins true, kitu atveju false. 
 ------------------------------------------------------------------------------------------------------ */
+
+class Movie {
+  constructor(title, director, budget) {
+    this.title = title;
+    this.director = director;
+    this.budget = budget;
+  }
+  wasExpensive() {
+    let budget = 100000000;
+    if (this.budget >= budget) {
+      this.budget = "true";
+    } else {
+      this.budget = "false";
+    }
```

Comment:
Turime raktinius žodelius `true` ir `false`, kurie yra boolean reikšmės, nereikia jų dėti į kabutes. Prie to paties, struktūrą

```js
if (sąlyga) {
  return true;
} else {
  return false;
}
```

galima supaprastinti į

```js
return sąlyga;
```